english/scream/scream.md

```diff
@@ -0,0 +1,13 @@
+Now you have screamed and it turns out to be that this is reality and you have
+to deal with the situation.
+
+You will do:
+
+[Jump out of joy and eat?](eat/jumpandeat.md)
+
+[Call your mom?](mom/eatwithmom.md)
+
+[Its right time to go for walk](walk/walkwiththought.md)
+
+[Keep calm and enjoy this?](http://www.gocomics.com/calvinandhobbes/)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.